app/src/main/java/com/keylesspalace/tusky/components/compose/ComposeViewModel.kt

```diff
@@ -213,7 +213,28 @@ class ComposeViewModel @Inject constructor(
         this.markMediaAsSensitive.value = this.markMediaAsSensitive.value != true
     }
 
-    fun didChange(content: String?, contentWarning: String?): Boolean {
+    fun handleCloseButton(contentText: String?, contentWarning: String?): ConfirmationPromptType {
+        return if (didChange(contentText, contentWarning)) {
+            when (composeKind) {
+                ComposeActivity.ComposeKind.NEW -> if (isEmpty(contentText, contentWarning)) {
```

Comment:
Nit: Please import `ComposeActivity.ComposeKind`, so the leading `ComposeActivity` can be dropped from these branches.